recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
+
+build:
+    number: 0
+    skip: true      # [win]
```

Comment:
Please add `detect_binary_files_with_prefix: true`. This *should* fix the `install_name_tool` issue.